service/src/test/java/bio/terra/landingzone/stairway/flight/create/CreateLandingZoneResourcesFlightIntegrationTest.java

```diff
@@ -0,0 +1,157 @@
+import bio.terra.stairway.exception.FlightNotFoundException;
+import bio.terra.stairway.exception.StairwayException;
+import java.time.Duration;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.retry.annotation.EnableRetry;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+@Tag("integration")
+@ActiveProfiles("test")
+@ExtendWith(SpringExtension.class)
+@SpringBootTest()
+@SpringBootApplication(
+    scanBasePackages = {
+      "bio.terra.common.logging",
+      "bio.terra.common.migrate",
+      "bio.terra.common.kubernetes",
+      "bio.terra.common.stairway",
+      "bio.terra.landingzone"
+    },
+    exclude = {DataSourceAutoConfiguration.class})
+@EnableRetry
+@EnableTransactionManagement
+public class CreateLandingZoneResourcesFlightIntegrationTest extends LandingZoneTestFixture {
+
+  @Mock private BearerToken bearerToken;
+
+  @Autowired LandingZoneService landingZoneService;
+
+  @Autowired
+  @Qualifier("landingZoneStairwayComponent")
+  StairwayComponent stairwayComponent;
+
+  UUID jobId;
+  UUID landingZoneId;
+  ProfileModel profile;
+  LandingZoneRequest request;
+  LandingZoneManager landingZoneManager;
+
+  @BeforeAll
+  static void init() {
+    Awaitility.setDefaultPollInterval(Duration.ofSeconds(5));
+  }
+
+  @BeforeEach
+  void setup() {
+
+    jobId = UUID.randomUUID();
+    landingZoneId = UUID.randomUUID();
+
+    profile =
+        new ProfileModel()
+            .managedResourceGroupId(resourceGroup.name())
+            .subscriptionId(UUID.fromString(azureProfile.getSubscriptionId()))
+            .tenantId(UUID.fromString(azureProfile.getTenantId()))
+            .cloudPlatform(CloudPlatform.AZURE)
+            .description("dummyProfile")
+            .id(UUID.randomUUID());
+    request =
+        new LandingZoneRequest(
+            StepsDefinitionFactoryType.CROMWELL_BASE_DEFINITION_STEPS_PROVIDER_TYPE.getValue(),
+            "v1",
+            Map.of(),
+            profile.getId(),
+            Optional.of(landingZoneId),
+            true);
+    landingZoneManager =
+        LandingZoneManager.createLandingZoneManager(
+            tokenCredential, azureProfile, resourceGroup.name());
+  }
+
+  @AfterEach
+  void cleanUp() throws Exception {
+    landingZoneManager.deleteResources(landingZoneId.toString());
```

Comment:
I am wondering if this cleanup [method ](https://github.com/DataBiosphere/terra-landing-zone-service/blob/d3fa7d060d40225b32de87aaa4705dd74a1a302e/service/src/test/java/bio/terra/landingzone/library/landingzones/LandingZoneTestFixture.java#L38)might be faster, where we delete the resource group altogether rather than iterating over each resource